src/tokenizer/stop_word_filter/mod.rs

```diff
@@ -160,7 +118,7 @@ impl<'a> TokenStream for StopWordFilterStream<'a> {
 
 impl Default for StopWordFilter {
     fn default() -> StopWordFilter {
-        StopWordFilter::english()
+        StopWordFilter::new(Language::English).unwrap()
```

Comment:
Can we remove the entire `StopWordFilter`?

Reply:
You mean feature-gate the entire `stop_word_filter` module and hence `StopWorldFilter` type behind `#[cfg(feature = "stopwords")]` (or maybe `#[cfg(feature = "stop-word-filter")]` then)?

Reply:
Personally, I'd say `StopWordFilter` can be useful without the built-in lists (For example, it can remove any list of words, not just stop words.) so I would suggest removing the `Default` impl (which I find questionable being tied to specific language in any case) and feature-gate `StopWorldFilter::new` so `StopWorldFilter::remove` could still be used with custom word lists.

Reply:
Sorry I mistyped! 
 I meant can we remove the default implementation for StopWordFilter.

I think we are on the same page here.
No more Default implementation.
Feature gate StopWordFilter::new.

Reply:
Indeed. Changed as suggested.